components/x-gift-article/src/HighlightRecipientMessage.scss

```diff
@@ -0,0 +1,60 @@
+@import '@financial-times/o-colors/main';
+@import '@financial-times/o-spacing/main';
+@import '@financial-times/o-typography/main';
+
+@include oTypography();
+
+.x-gift-article__highlight-recipient-message {
+    position: relative;;
+    background-color: oColorsByName('slate');
+    color: oColorsMix('ft-grey', 'white', 20);
+    padding: oSpacingByName('s3');
+  
+    .x-gift-article__highlight-recipient-message__title {
```

Comment:
**Change request:** The classnames in this file look a bit like [BEM](https://getbem.com/naming/) but seems to be suggesting multiple elements levels. I think it might be useful to refactor this to follow the BEM naming